### ISSUE TITLE
ODT->PDF Conversion: Header/Footer table column spacing is ignored in PDF

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#192]
```
>>> What steps will reproduce the problem?
1. Convert the attached "header_footer_columns.odt" file to PDF using the 
PdfConverter XDocReport class.

>>> What is the expected output? What do you see instead?
The expected output should be the header and footer on the PDF having the same 
column spacing as the ODT file. The PDF just has equal spacing.

>>> What version of the product are you using? On what operating system?
1.0.0-SNAPSHOT, latest code from the git repo (as of this morning.)
Windows 7 64-bit
```

Original issue reported on code.google.com by `PhilipDN...@gmail.com` on 9 Nov 2012 at 7:24

Attachments:
- [header_footer_columns.odt](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-192/comment-0/header_footer_columns.odt)
- [header_footer_columns.pdf](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-192/comment-0/header_footer_columns.pdf)

**Comment by GoogleCodeExporter** [comment]
```
@Leszek : I have add a JUnit 
http://code.google.com/p/xdocreport/source/browse/thirdparties-extension/org.odf
toolkit.odfdom.converter.pdf/src/test/resources/org/odftoolkit/odfdom/converter/
core/Issue192.odt with the odt attached of this issue.
```

Original comment by `angelo.z...@gmail.com` on 10 Nov 2012 at 2:10
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 1.0.0
```

Original comment by `abe...@gmail.com` on 13 Dec 2012 at 8:05
- Changed state: **Fixed**
